### ISSUE TITLE
PrepExternalSchema blastp error

### ISSUE THREAD

**Issue by cizydorczyk** [B-UMMI/chewBBACA#105]
Hello,

When trying to prep an external schema, I get the following error:

```
Error on adapt_loci:
Traceback (most recent call last):
  File "/home/conrad/anaconda3/envs/chewbbaca-2.7.0/lib/python3.9/site-packages/CHEWBBACA/utils/multiprocessing_operations.py", line 39, in function_helper
    results = input_args[-1](*input_args[0:-1])
  File "/home/conrad/anaconda3/envs/chewbbaca-2.7.0/lib/python3.9/site-packages/CHEWBBACA/PrepExternalSchema/PrepExternalSchema.py", line 362, in adapt_loci
    raise ValueError(blast_stderr)
ValueError: ['Warning: [blastp] To obtain better run time performance, please run blastdb_aliastool -seqid_file_in <INPUT_FILE_NAME> -seqid_file_out <OUT_FILE_NAME> and use <OUT_FILE_NAME> as the argument to -seqidlist']
```

I am using the latest version of chewBBACA. The schema files are named "locus_number.fasta", each containing alleles for each locus.

Any help is appreciated.

Thank you,
Conrad

**Comment by cizydorczyk** [comment]
fixed by installing blast+ 2.9.0

**Comment by rfm-targa** [comment]
Hello @cizydorczyk,

Thank you for you interest in chewBBACA. It's good to know that you could solve the issue by installing BLAST 2.9.0. chewBBACA works best with that BLAST version. Also, updating chewBBACA from v2.7.0 to v2.8.5 will add new features and important bugfixes, including a bugfix for that BLAST warning that would halt the PrepExternalSchema process (you can read more about the changes since v2.7.0 in our [Changelog](https://github.com/B-UMMI/chewBBACA/blob/master/CHANGELOG.md)).
Please don't hesitate to contact us if there's anything else.

Rafael
